### PR TITLE
Update exclude paths for output docs

### PR DESCRIPTION
**Pull request by dedemorton**
Updates the Beats documentation to pull output doc source in 7.x and 7.5 from libbeat/outputs

### CODE REVIEW

**Comment by dedemorton** [comment]
@nik9000 I've added yaml anchors (following your example). I think it makes the conf.yaml easier to read, but seems a bit less precise. I'm OK with that, tho.

File: conf.yaml [hunk 1]
```diff
@@ -945,11 +945,11 @@ contents:
               -
                 repo:   docs
                 path:   shared/versions/stack/{version}.asciidoc
-                exclude_branches:   [ 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 1.3, 1.2, 1.1, 1.0.1 ]
+                exclude_branches:   &sharedfileexcludes [ 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 1.3, 1.2, 1.1, 1.0.1 ]
```
Comment (nik9000) [inline conf.yaml]: Probably best to name it something like `beatsSharedExclude` or something like that. These names are shared by all of our projects.

File: conf.yaml [hunk 2]
```diff
@@ -1009,19 +1009,19 @@ contents:
               -
                 repo:   beats
                 path:   libbeat/processors/*/docs/*
-                exclude_branches:   [ 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 1.3, 1.2, 1.1, 1.0.1 ]
+                exclude_branches:   &processorexcludes [ 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 1.3, 1.2, 1.1, 1.0.1 ]
```
Comment (nik9000) [inline conf.yaml]: Same comment above.

File: conf.yaml [hunk 3]
```diff
@@ -1009,19 +1009,19 @@ contents:
...
               -
                 repo:   beats
                 path:   libbeat/outputs/*/docs/*
-                exclude_branches:   [ 7.x, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 1.3, 1.2, 1.1, 1.0.1 ]
+                exclude_branches:   &outputexcludes [ 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 1.3, 1.2, 1.1, 1.0.1 ]
```
Comment (nik9000) [inline conf.yaml]: Same comment as above.

File: conf.yaml [hunk 4]
```diff
@@ -1047,7 +1047,7 @@ contents:
               -
                 repo:   beats
                 path:   x-pack/libbeat/docs
-                exclude_branches:   [ 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 1.3, 1.2, 1.1, 1.0.1 ]
+                exclude_branches:   &xpacklibbeatexcludes [ 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 1.3, 1.2, 1.1, 1.0.1 ]
```
Comment (nik9000) [inline conf.yaml]: Same comment as above.

**Pull request merged**